auth-web/src/util/common-util.ts

```diff
@@ -255,4 +255,8 @@ export default class CommonUtils {
     const stringIndexValue: string = index.toString()
     return stringIndexValue.padStart(2, '0')
   }
+
```

Comment:
Can we do it with Css?
text-transform: capitalize;

Reply:
yup, I'll fix